src/app/content/highlights/components/HighlightStyles.tsx

```diff
@@ -63,10 +64,12 @@ export const Modal = styled.div`
 // tslint:disable-next-line:variable-name
 export const Wrapper = styled.div`
   z-index: 1;
-  width: 100%;
+  max-width: ${contentWrapperMaxWidth}rem;
   background: ${theme.color.neutral.base};
-  margin: 3rem;
+  margin: 7.2rem 2.4rem 2.4rem;
```

Comment:
this isn't centering the modal on large screens

Reply:
added a justify-content: center; to the Mask